api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -375,7 +375,6 @@ public void getAge_shouldGetAgeAfterDeath() {
 		deathDate.set(2000, Calendar.JUNE, 3);
 		Person person = new Person();
 		person.setBirthdate(birthdate.getTime());
-		person.setDead(true);
```

Comment:
Coz it's set now automatically, when you set deathDate. So it looks like a just unneeded line of code.